src/jspm-hmr-server.ts

```diff
@@ -66,7 +66,7 @@ export function createServer(options: ServerOptions): JspmHmrServer {
 
     const proxy = httpProxy.createProxyServer();
     app.use(proxyRoute, (req, res) => {
-      req.url = req.baseUrl;
+      req.url = `${req.baseUrl}${req.url}`;
```

Comment:
because `proxyRoute` option param above is selecting mountpath for proxy, so I'm wondering if that is relevant in this case